Library/Formula/docker-swarm.rb

```diff
@@ -32,6 +32,6 @@ def install
 
   test do
     output = shell_output(bin/"docker-swarm --version")
-    assert output.include? "swarm version 0.2.0 (HEAD)"
+    assert output.include? "swarm version 0.3.0 (HEAD)"
```

Comment:
This is why the tests failed:
```        Error: 1 problem in 1 formula
docker-swarm:
 * Use `assert_match` instead of `assert ...include?`
```
It's a small thing, but instead we like to do `assert_match "swarm version #{version}", output`. It makes assertion failures easier to understand. Thanks!